addons/web/static/src/core/dialog/dialog.scss

```diff
@@ -1,52 +1,13 @@
-.o_dialog {
-  > .modal {
-    display: block;
-  }
-}
-
-.modal.o_technical_modal {
-  .modal-content {
-    border-radius: 0;
-
-    .modal-header .modal-title {
-      word-break: break-word;
-    }
-
-    .modal-footer {
-      flex-wrap: wrap;
-      text-align: left;
-      justify-content: flex-start;
-
-      footer {
-        > :not(:first-child) {
-          margin-left: 0.25rem;
-        }
-        > :not(:last-child) {
-          margin-right: 0.25rem;
+.modal.o_technical_modal .modal-footer {
+    button {
+        margin: map-get($spacers, 0) map-get($spacers, 1) map-get($spacers, 1);
+  
+        &:last-child {
+          margin-right: map-get($spacers, 0);
```

Comment:
Think about RTL... for inline elements use `maring-inline[start/end]` css properties